pkg/apis/playlist/v0alpha1/register.go

```diff
@@ -41,10 +45,11 @@ func (b *PlaylistAPIBuilder) GetGroupVersion() schema.GroupVersion {
 }
 
 func (b *PlaylistAPIBuilder) InstallSchema(scheme *runtime.Scheme) error {
-	err := AddToScheme(scheme)
-	if err != nil {
-		return err
-	}
+	scheme.AddKnownTypes(SchemeGroupVersion,
+		&Playlist{},
+		&PlaylistList{},
+	)
+	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
 	return scheme.SetVersionPriority(SchemeGroupVersion)
 }
```

Comment:
aha, this is interesting, what does it do? Does it add playlist schemas to some kind of repository that is used (or to be used) for objects validation?

Reply:
in k8s, you tell the global schema what golang classes belong to the group+version... and it does a **_lot_** of magic in linking openapi specs to expected behavior.  TBH, i am not totally clear all the implications, but it is required or things won't start.

Reply:
fair enough!